.github/workflows/upgrade.yml

```diff
@@ -26,11 +26,11 @@ jobs:
           ./gradlew run --args='..'
 
       - name: Generate Github Token
-        uses: tibdex/github-app-token@v2
```

Comment:
The old one is deprecated, see

https://github.com/tibdex/github-app-token
https://github.com/actions/create-github-app-token